src/battle/star/up_and_away/7952E0.c

```diff
@@ -43,23 +43,26 @@ ApiStatus func_802A15B4_795894(Evt* script, s32 isInitialCall) {
     return ApiStatus_BLOCK;
 }
 
-#ifdef NON_EQUIVALENT
 ApiStatus func_802A1628_795908(Evt* script, s32 isInitialCall) {
     Actor* targetActor = get_actor(get_actor(script->owner1.actorID)->targetActorID);
-    s32 flags = targetActor->flags;
-
+    u32 flags;
```

Comment:
u32 flags = ...

Reply:
Per previous conversation, assignment has to be in this order